app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,40 @@
+class ReviewsController < SecuredController
+  skip_before_action :authorize_request, only: %i[index show]
+
+  # GET /reviews
+  def index
+    user_sub = request.headers['body']
+    fab = User.find_by(user_sub: user_sub)
+    review = fab.reviews
+    render json: review
+  rescue ActiveRecord::RecordNotFound
```

Comment:
Perhaps this sort of error handling is a bit too much for what we want to do. 

We only use a rescue clause if the particular failure will result in an exception. In the case here, if a resource cannot be found then Rails will simply return a 404 error (and not an exception).